scripts/start.js

```diff
@@ -98,6 +100,19 @@ browserSync.init({
             fn: _.debounce(generateApis, 1500),
         },
 
+        {
+            match: [
+                lintStylesGlob,
+            ],
+            fn: _.debounce(() => {
+                let _cmd = `stylelint --color "${lintStylesGlob}"`;
+                let _proc = exec(_cmd, { cwd: CONFIG.root });
+
+                _proc.stdout.pipe(process.stdout);
+                _proc.stderr.pipe(process.stderr);
```

Comment:
It's not the prettiest way to do it, but we can refactor it later with new pipeline functionality.

Reply:
eeek